### PR TITLE
raxml-ng 0.7.0

### PR DESCRIPTION
**Pull request by jonchang**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/raxml-ng.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class RaxmlNg < Formula
   desc "RAxML Next Generation: faster, easier-to-use and more flexible"
   homepage "https://sco.h-its.org/exelixis/web/software/raxml/"
   url "https://github.com/amkozlov/raxml-ng.git",
-    :tag => "0.5.1",
-    :revision => "8a3d6af12fbff60239744595631a468275d5a02a"
+    :tag      => "0.7.0",
```
Comment (sjackman) [inline Formula/raxml-ng.rb]:
```suggestion
    :tag => "0.7.0",
```
Please remove the additional white space.

**Comment by sjackman** [comment]
`brew test raxml-ng` is failing. I don't see any evident error message.
https://circleci.com/gh/brewsci/homebrew-bio/4825
@jonchang Are you able to reproduce this test failure locally or in Docker?

File: Formula/raxml-ng.rb (hunk 1 above)
Comment (jonchang) [reply Formula/raxml-ng.rb]: Audit demands this:

```
brewsci/bio/raxml-ng:
  * C: 5: col 5: Align the elements of a hash literal if they span more than one line.
```

**Comment by jonchang** [comment]
Yeah, I dunno why it was failing, but it seems to work with a different test.

**Comment by jonchang** [comment]
Merged. Thanks Shaun!

**Pull request closed without merging**